include/zen/display-system.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ZEN_DISPLAY_SYSTEM_H
+#define ZEN_DISPLAY_SYSTEM_H
+
+#include <wayland-server-core.h>
+#include <zen-desktop-protocol.h>
+
+struct zn_display_system_switch_event {
+  enum zen_display_system_type type;
+};
+
+struct zn_display_system {
+  struct wl_global* global;
+
+  struct wl_list resources;
```

Comment:
```suggestion
  struct wl_list resources; // wl_resource::link
```
(It's almost obvious.)

Reply:
I don't think there is any confusion without comment.